### DISCUSSION TITLE
Include a Donation Button!

### DISCUSSION THREAD

**Discussion by jeff-restlesslabs** [tprouvot/Salesforce-Inspector-reloaded#336] [Ideas]
SF Inspecotor is hands down the best tool for SF Admins, Developers, etc.  
I cannot count how many times I use this per day to run exports, analyze fields and imports.
I would love to be able to repay the team for the amount of time this has saved me.

**Comment by megmoo** [comment]
I would definitely donate. This tool has saved me so many hours and increased my productivity immensely. I'm grateful for all the work and the fact that it's free.

**Comment by jeff-restlesslabs** [comment]
Please upvote this thread!

**Comment by tprouvot** [comment]
Hi @jeff-restlesslabs, @megmoo 👋 

Thank you very much for requesting the ability to donate for the extension, you definitely helped on the decision to open for donation! 

The donate button will be available in the extension by the next release 1.26 (beta soon) and directly from today in the [sponsor page](https://github.com/sponsors/tprouvot).
